includes/omise-wc-setting.php

```diff
@@ -0,0 +1,100 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+return array(
+	'enabled' => array(
+		'title'       => __( 'Enable/Disable', $this->gateway_name ),
+		'type'        => 'checkbox',
+		'label'       => __( 'Enable Omise Payment Module.', $this->gateway_name ),
+		'default'     => 'no'
+	),
+	'sandbox' => array(
+		'title'       => __( 'Sandbox', $this->gateway_name ),
```

Comment:
@nimid ✋ have a question here, would be better to not split this setting array out from the `omise-wc-gateway.php` file since this array still referring some data from `$this`.
what do you think?